hal/halx86/CMakeLists.txt

```diff
@@ -54,13 +54,15 @@ if(ARCH STREQUAL "i386")
     include(legacy.cmake)
     include(up.cmake)
     include(pic.cmake)
+    include(xbox.cmake)
```

Comment:
Do we need to build the xbox hal in all cases? Or only when we configured a build with SARCH set to XBOX?

Reply:
I think it is better, because we can just select it in setup, and it will be checked by builders and CI automatically. Also it simplifies Xbox port testing a lot.

Reply:
Ok then :)